ci/rpm_test_daos_test.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -uex
+
+# DAOS_PKG_VERSION environment variable needs to be set for this script
+
+if git show -s --format=%B | grep "^Skip-test: true"; then
+  exit 0
+fi
+nodelist=("${NODELIST//,/ }")
+src/tests/ftest/config_file_gen.py -n "${nodelist[0]}" \
+  -a /tmp/daos_agent.yml -s /tmp/daos_server.yml
+src/tests/ftest/config_file_gen.py -n "${nodelist[0]}" -d /tmp/dmg.yml
+scp -i ci_key /tmp/daos_agent.yml jenkins@"${nodelist[0]}":/tmp
```

Comment:
nit-pick, but these three scp could all be one on one line as they're all be copied to the same place.

Reply:
I am going to have follow up PRs that are going to be further refactoring the test stages, Can this wait until one of those?